.github/workflows/publish-rule-book.yaml

```diff
@@ -0,0 +1,114 @@
+name: Publish Rule Book
+
+on:
+  # push:
+  #   branches:
+  #     - main
+  pull_request:
+    branches: [main]
+
+env:
+  BOARD_GAME_GEEK_RULE_BOOK: https://boardgamegeek.com/filepage/272787
+  PO4A_VERSION: "0.71"
+  PO4A_GH_URL: https://github.com/mquinson/po4a/releases/download
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [en, pl]
```

Comment:
This line seems redundant, unless you specifically want to have elements in the matrix without Wersja

Reply:
Fixed